### PR TITLE
Refactor: Numbered section

### LINKED ISSUE #101: Homepage - Implement Industry Standards + numbered items + border image
image

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #101 

Thanks to https://stackoverflow.com/questions/5732836/html-css-numbered-list-with-numbers-inside-of-circles

## What this PR does

Everything between the purple dots, `Bringing industry standards to California’s transit providers` and the purple/pink dots.

- [x] refactor picture/image HTML with `<ul>` `<ol>` and CSS pseudoelement circles
- [x] delete unused images
- [x] delete unused css
- [x] update copy
- [x] update image border
- [x] use correct colors
- [x] reorders the big 2 sections so the homepage is now all in the correct order 
- [x] re-do the spacing below/above the purple dots, so that it is even 
- [x] declare mobile-first padding between the two text-only sections and the dots 

image
image

image

### CODE REVIEW

**Comment by machikoyasuda** [comment]
This is now ready for review @angela-tran @thekaveman

**Comment by thekaveman** [comment]
Wait I thought `.constricted` was removed in #133 but it is back here?

File: src/index.html [hunk 1]
```diff
@@ -57,17 +123,15 @@ <h3>Enabling contactless payments</h3>
...
+                                                                                                                                                                                                                                                                                                                                                                        transportation providers to purchase contactless payments hardware and software directly from vendors without further
+                                                                                                                                                                                                                                                                                                                                                                        competitive bidding. Learn about the MSAs in our
```
Comment (thekaveman) [inline src/index.html:134]: Looks like some extra whitespace got in here

**Comment by machikoyasuda** [comment]
@thekaveman Oh my bad. Painful rebase.

File: src/index.html [hunk 2]
```diff
@@ -78,31 +142,33 @@ <h3>Enabling contactless payments</h3>
...
+          (MST), which offers a half-price Senior Fare. Now older adults (65+) who are able to
+          <a href="https://login.gov/help/verify-your-identity/how-to-verify-your-identity/#requirements-for-identity-verification" target="_blank">electronically verify their identity</a>
           are able to access MST's reduced fares without the hassle of paperwork.
         </p>
         <p>
           We worked with state partners on this product launch, and next we're working to bring youth, lower-income riders,
           veterans, people with disabilities, and others the same instant access to free or reduced fares across all California
-          transit providers, without having to prove eligibility to each agency.
+                                                                                                                                                                                                                                                                                                                                                                        transit providers, without having to prove eligibility to each agency.
```
Comment (thekaveman) [inline src/index.html:171]: Whitespace

File: src/index.html [hunk 3]
```diff
@@ -114,110 +180,32 @@ <h3>Automating customer discounts</h3>
...
-        />
+          width="106" />
       </picture>
       <section class="right-callout">
         <h3>Standardizing information for easy trip planning</h3>
         <p>
           Cal-ITP is helping transit providers remove the guesswork for riders wondering when the next bus or train will arrive or
           if they’ll make their connection by using the General Transit Feed Specification (GTFS)—the global standard for
-          publishing transit information. Cal-ITP developed
+                                                                                                                                                                                                                                                                                                                                                                        publishing transit information. Cal-ITP developed
```
Comment (thekaveman) [inline src/index.html:190]: Whitespace

File: src/index.html [hunk 4]
```diff
@@ -114,110 +180,32 @@ <h3>Automating customer discounts</h3>
...
-            href="https://dot.ca.gov/cal-itp/california-minimum-general-transit-feed-specification-gtfs-guidelines"
-            >California Minimum GTFS Guidelines</a
-          >
+            href="https://dot.ca.gov/cal-itp/california-minimum-general-transit-feed-specification-gtfs-guidelines">California Minimum GTFS Guidelines</a>
           and is working to ensure statewide GTFS static coverage by the end of 2020 and GTFS Realtime by the end of 2021. Along
           the way, the Cal-ITP team will support transit providers by assessing their systems and providing technical assistance
-          so riders can easily access complete, accurate, consistent, and timely mobility data for their journey.
+                                                                                                                                                                                                                                                                                                                                                                        so riders can easily access complete, accurate, consistent, and timely mobility data for their journey.
```
Comment (thekaveman) [inline src/index.html:197]: Whitespace

**Review by thekaveman — CHANGES_REQUESTED** [review]

**Comment by thekaveman** [comment]
@mrose914 can you please review the content of this numbered section: https://deploy-preview-136--cal-itp-website.netlify.app/

We used the exact wording from the content doc, just want to make sure this is the intended casing:

image

**Comment by machikoyasuda** [comment]
@thekaveman Could we still merge this w/o waiting on final copy? I can make a new Issue for the copy issue on just the numbered area. This PR moves the giant <section>s around and it's going to be helpful for subsequent PRs (or an enormous rebase headache).

**Review by thekaveman — APPROVED** [review]

**Pull request merged**